server/utils/validations/admin/admin.ts

```diff
@@ -0,0 +1,17 @@
+import { param } from "express-validator";
+import { ERROR_MESSAGES } from "./constants";
+import { validate } from "../../../middleware/validate";
+
+/*
+  Validators
+  - 각 입력 값에 대한 유효성 검사
+*/
+const userIdValidator = param("userId")
+	.isNumeric()
```

Comment:
nit: 유저 ID가 양의 정수이므로 `.isInt({ min: 1 })`을 사용하면 더 좋을 것 같습니다.
```suggestion
	.isInt({ min: 1 })
```